src/engine/sprite.cxx

```diff
@@ -6,64 +6,62 @@
 #include "basics/isoMath.hxx"
 #include "basics/log.hxx"
 
-Sprite::Sprite(Point isoCoordinates) : _isoCoordinates(isoCoordinates)
+Sprite::Sprite(Point isoCoordinates) : isoCoordinates(isoCoordinates)
```

Comment:
I think that member variable name and argument name should different.

Reply:
oh... totally overlooked that while auto-renaming... fixed